src/nfa/nfa.cc

```diff
@@ -860,30 +886,23 @@ void Vata2::Nfa::intersection(Nfa *res, const Nfa &lhs, const Nfa &rhs, ProductM
     }
 }
 
-/*
-Util::BinaryRelation Nfa::computeSimulation() const {
-    VATA::ExplicitLTS LTSforSimulation;
-    Symbol maxSymbol = 0;
-    for (State stateFrom = 0; stateFrom < transitionrelation.size(); ++stateFrom) {
-        for (const TransSymbolStates &t : getTransitionsFromState(stateFrom)) {
-            for (State stateTo : t.states_to) {
-                LTSforSimulation.add_transition(stateFrom, t.symbol, stateTo);
-            }
-            if (t.symbol > maxSymbol) {
-                maxSymbol = t.symbol;
-            }
-        }
+Vata2::Util::BinaryRelation Vata2::Nfa::compute_relation(const Nfa& aut, const StringDict& params) {
+    if (!haskey(params, "relation")) {
```

Comment:
Suggestion: add default to the params `{'relation', 'simulation'}` or whatever it is called. (See other `StringDict& params` usages.)